### ISSUE TITLE
Support filtering based on tags

### ISSUE THREAD

**Issue by biggianteye** [biggianteye/book-chooser#4]
Choosing the next highest rated book is great, but sometimes you want something specific.

Should be able to filter books that have a specific tag e.g.

```
--include genre:sci-fi
```

Should be to able to filter out as well. eg.

```
--exclude format:anthology
```

Should also be able to specify multiple tags. eg.
```
--include genre:dystopian --include setting:space --exclude format:novella
```

Exclusions should take precedence over inclusions.

Question: Should multiple filters of the same type be combined using `AND` or `OR`? Should this be stated explicitly?

**Comment by biggianteye** [comment]
Fixed in #8 and #17.

**Comment by biggianteye** [comment]
> Question: Should multiple filters of the same type be combined using `AND` or `OR`? Should this be stated explicitly?

I used `AND`. If I come across a need for `OR`, I'll revisit this.
